Added citation file (#2)
CardiacModelling/Gamma_0: README.md

# A parameter representing missing charge should be considered when calibrating action potential models

This code is associated with the paper:
 * Barral et al. "A parameter representing missing charge should be considered when calibrating action potential models" [10.3389/fphys.2022.879035](https://doi.org/10.3389/fphys.2022.879035).
   

> If you use the code in further academic work please cite this paper, using the information in the [CITATION file](./CITATION).

Author: ```suggestion
To cite this work, please use the information in the [CITATION file](./CITATION).
```
